traefik.go

```diff
@@ -114,13 +137,19 @@ Complete documentation is available at https://traefik.io`,
 
 	traefikConfiguration.ConfigFile = toml.ConfigFileUsed()
 
-	kv, err := CreateKvSource(traefikConfiguration)
+	kv, err = CreateKvSource(traefikConfiguration)
 	if err != nil {
 		fmtlog.Println(err)
 		os.Exit(-1)
 	}
 
-	if kv != nil {
+	usedCmd, err := f.GetCommand()
+	if err != nil {
+		fmtlog.Println(err)
+		os.Exit(-1)
+	}
+	// IF a KV Store is enable and no sub-command called in agrs
+	if kv != nil && usedCmd == traefikCmd {
```

Comment:
If the sub-command `storeconfig` is called, we want to parse all sources (flæg and toml) except the KV source (prevent update issues). 
Maybe checking the called command is not beautiful; but I d'ont see any other way so far :sweat_smile: